server.go

```diff
@@ -111,6 +117,14 @@ func parseReq(buff []byte) (quoteRequest, error) {
 }
 
 func makeResp(req quoteRequest) quoteResponse {
+	// Only use the first 3 char of a stock
+	var truncatedStock string
+	if stockLen := len(req.stock); stockLen < 3 {
```

Comment:
Would have liked to use something like
```go
truncateLen := math.Min(3, len(req.stock))
...
stock: strings.ToUpper(resp.stock[:truncateLen])
```
but the [docs](https://golang.org/pkg/math/#Min) say that `math.Min` only works with `float`s and 😛 to type conversions.